src/components/Nav.css

```diff
@@ -1,20 +1,24 @@
 nav {
   height: 50px;
   width: 100%;
-  position: absolute;
-  left: 0;
-  bottom: 0;
-  right: 0;
   text-align: center;
   background-color: white;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
 }
 
 ul {
   list-style: none;
   padding: 0;
+  width: 100%;
```

Comment:
The default `width` for block-level elements is `100%`. Was this not working as-expected without this here?

Reply:
It works fine without